DigitalLearningSolutions.Web/ViewModels/LearningMenu/SectionCardViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Data.Models.CourseContent;
+
+    public class SectionCardViewModel
+    {
+        public readonly string Title;
+        public readonly bool HasLearning;
+        public readonly double PercentComplete;
```

Comment:
Do these need to be public, I think only the title needs to be public? In particular the percent complete should not be public because we want the method to be used to get the percentage complete.

And for the field(s) which need to be public I'd suggest using properties with a public getter and no setter, just for consistency 🙂